lib/buttercms-ruby.rb

```diff
@@ -1,9 +1,10 @@
 require 'json'
 require 'ostruct'
 require 'logger'
+require 'uri'
+require 'net/http'
 
-require "buttercms/errors"
-
+require_relative "buttercms/errors"
```

Comment:
`require_relative` should not be use here because these files should be in the load path already.